extensions/resource-deployment/src/ui/deployAzureSQLDBWizard/pages/azureSettingsPage.ts

```diff
@@ -320,13 +308,9 @@ export class AzureSettingsPage extends BasePage {
 		let url = `https://management.azure.com/subscriptions/${this.wizard.model.azureSubscription}/providers/Microsoft.Sql/servers?api-version=2019-06-01-preview`;
 		let response = await this.wizard.getRequest(url);
 		if (response.data.value.length === 0) {
+			this.wizard.showErrorMessage(localize('deployAzureSQLDB.NoServerLabel', "No servers found, please select another subscription"));
```

Comment:
So in addition to displaying this message, we need to ensure that the Wizard controls locked up and the user cannot continue the wizard unless corrective action is taken. I do not see any code here that will disable buttons, so I worry that we are not doing the correct thing in the face of this error condition of getting no subscription information.